### DISCUSSION TITLE
linter checks too slow, run `golangci-lint run --new=false --new-from-rev=HEAD~1 --fix=false --whole-files=false -v` 6 min

### DISCUSSION THREAD

**Discussion by cold-bin** [golangci/golangci-lint#4780] [Q&A]
the cmd `golangci-lint run --new=false --new-from-rev=HEAD~1 --fix=false --whole-files=false -v` will scan all project code?

my project has 50,0000 go lines

**Comment by cold-bin** [comment]
can it specify a folder or file for analysis

**Comment by ldez** [comment]
I [asked you to stop spamming discussions](https://github.com/golangci/golangci-lint/discussions/4776#discussioncomment-9605759): you will be banned for 7 days.
